atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/atomic/SimpleCommitTimestampAtomicTableTest.java

```diff
@@ -28,53 +28,51 @@
 import com.palantir.atlasdb.transaction.encoding.TransactionStatusEncodingStrategy;
 import com.palantir.atlasdb.transaction.encoding.V1EncodingStrategy;
 import com.palantir.atlasdb.transaction.service.TransactionStatus;
-import java.util.Arrays;
-import java.util.Collection;
+import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ExecutionException;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
-/* TODO(boyoruk): Migrate to JUnit5. */
-@RunWith(Parameterized.class)
 public class SimpleCommitTimestampAtomicTableTest {
-    @Parameterized.Parameter
-    public TransactionStatusEncodingStrategy<TransactionStatus> encodingStrategy;
 
-    @Parameterized.Parameters
-    public static Collection<Object[]> parameters() {
-        return Arrays.asList(new Object[][] {{V1EncodingStrategy.INSTANCE}, {TicketsEncodingStrategy.INSTANCE}});
+    public static List<TransactionStatusEncodingStrategy<TransactionStatus>> encodingStrategies() {
+        return List.of(V1EncodingStrategy.INSTANCE, TicketsEncodingStrategy.INSTANCE);
     }
 
     private AtomicTable<Long, TransactionStatus> atomicTable;
```

Comment:
I don't think this needs to exist as a private final field anymore. Can be converted to a local variable in each method.